custom/icds_reports/tasks.py

```diff
@@ -14,13 +14,14 @@
 
 
 @periodic_task(run_every=crontab(minute=0, hour=21), acks_late=True, queue='background_queue')
-def move_ucr_data_into_aggregation_tables(date=None):
+def move_ucr_data_into_aggregation_tables(date=None, intervals=3):
     date = date or datetime.utcnow().date()
-    now = datetime.utcnow().date()
+    monthly_date = date.replace(day=1)
+    now = datetime.utcnow().date().replace(day=1)
```

Comment:
Why does `now` have `replace(day=1)` at the end. this changes the definition of these aggregations and means that `now == monthly_date` which I don't think is desired